statsmodels/tools/tests/test_decorators.py

```diff
@@ -40,3 +41,14 @@ def f(self):
         ex.b = -1
 
     assert_equal(ex._cache, dict(b=1,))
+
+
+@pytest.mark.xfail(reason="scipy.stats raises at import-time with -OO")
```

Comment:
How sure are you that scipy.stats raises?  I tried locally and have seen no issues.

Reply:
Looks like it's a scipy version issue.  It raises in 1.1.0.  After upgrading to 1.2.1, it no longer raises.  I'll change xfail to xfailif